src/styles/blocks/header.scss

```diff
@@ -0,0 +1,12 @@
+.header {
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+  justify-content: flex-end;
+  box-sizing: border-box;
+  padding: 0 20px 40px;
+  background-image: url(../../../src/images/photos/1.jpg);
```

Comment:
Check the path, there is no background image at your demo.
Also, you can use a shorthand for background property here.